File: back-end/Proyecto.Domain/Entities/GrupoGraduado.cs
using System;

namespace Proyecto.Domain.Entities
{
    public class GrupoGraduado
    {
        public int Id { get; set; }
        public DateTime Anyo { get; set; }
        public string Sexo { get; set; }
        public int CursoId { get; set; }
        public int? Cantidad { get; set; }

        public Curso Curso { get; set; }
    }
}

File: front-end/src/app/models/GrupoGraduado.ts
import { CursoResponse } from './Curso';

export interface GrupoGraduadoResponse {
	id: number;
	anyo: Date;
	sexo: string;
	cantidad?: number;
	curso: CursoResponse;
}

export interface GrupoGraduadoRequest {
	anyo: Date;
	sexo: string;
	cursoId: number;
	cantidad?: number;
}
